spyder/widgets/editor.py

```diff
@@ -1690,8 +1780,6 @@ def save_copy_as(self, index=None):
             txt = to_text_string(finfo.editor.get_text_with_eol())
             try:
                 finfo.encoding = encoding.write(txt, filename, finfo.encoding)
-                self.file_saved.emit(str(id(self)), index, filename)

```

Comment:
Why is this not needed anymore?

Reply:
@ccordoba12   The `copy as` function creates a new file with the same information as the current index, but it doesn't make any changes to the current index or current filename (in the tab or the self.data). The `file_saved.emit` was updating the other split panels for the current index/filename, but this isn't necessary since it's not changed at all.

It's simple to recreate to see what's happening, even without moving tabs.
1.  Open one file in one panel.
2.  Split horizontally or vertically.
3.  Click `save_copy_as` and save the file under a new name.  You'll see the new file open in both panels, so there are now 2 tabs.
4.  Click focus onto the other panel.
5.  Notice that both tabs get updated with the same file name and no longer match the first panel.

This one really isn't related to moving tabs, but I found it while testing my changes.  If you want, I can move it to a separate PR.

Reply:
Don't worry, it's really good that you have detected and solved this problem as part of this PR.